packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -351,6 +354,31 @@ class MediaQueryData {
   /// gesture behavior over the framework constants.
   final DeviceGestureSettings gestureSettings;
 
+  /// {@macro dart.ui.ViewConfiguration.displayFeatures}
+  ///
+  /// See also:
+  ///
+  ///  * [dart:ui.DisplayFeatureType], where you can see types of display features.
+  ///  * [dart:ui.DisplayFeatureState], where you can see possible states for
+  ///  folding features ([dart:ui.DisplayFeatureType.fold] and
+  ///  [dart:ui.DisplayFeatureType.hinge])
+  final List<ui.DisplayFeature> displayFeatures;
+
+
+  /// Area of the display that is obstructed specifically by the [DisplayFeatureType.hinge].
+  ///
+  /// A hinge is the space between 2 physical displays. The size of the Flutter view
+  /// in this case contains both screens and the area between them. The hinge
+  /// area can be used to separate the layout into 2 logical areas or panels in
+  /// the application.
+  ui.DisplayFeature? get hinge {
```

Comment:
Why does the hinge get special treatment over other display features? What if a future phone introduces three (or more hinges)?

Reply:
The `hinge` is the only one of the three that forces apps to adapt. The `fold` does not stop pixels from rendering and the `cutout` is easily negotiated using SafeArea.
The initial reasoning was that having a nullable `hinge` property would make it easier for developers, but I now see it wasn't used in the rest of the PRs. 

Since the reasoning for having it is not rock solid, would you prefer if I remove it?

Reply:
I would just remove it then.